lib/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  env: {
+    browser: true
```

Comment:
`browser: true` should just be added to the `env` in the root .eslintrc file, and this file should be removed.